docs/src/standalone/pages/vegetation/canopy_structure/prescribed_structure.md

```diff
@@ -0,0 +1,6 @@
+# Canopy Structure (Prescribed)
+
+The canopy structure is prescribed, and the following states are read from maps:
```

Comment:
Currently only rooting depth and total leaf area index are read in from maps. SAI = 0 by default, and RAI and canopy height are scalars also.

Reply:
We might want to add these from maps next.